src/components/story/story.service.ts

```diff
@@ -86,27 +89,14 @@ export class StoryService {
     dto: UnsecuredDto<Story>,
     session: Session,
   ): Promise<Story> {
-    const securedProps = await this.authorizationService.secureProperties(
-      Story,
-      {
+    return {
+      ...dto,
+      ...this.privileges.for(session, Story).secure({
         ...dto,
         scriptureReferences: this.scriptureRefs.parseList(
-          dto.scriptureReferences,
+          dto.scriptureReferences as unknown as DbScriptureReferences,
```

Comment:
These should be changed with the same strategy we used in products. Since we changed the Producible DTO, that affects all these as well.

Reply:
Let me work on it again. This threw lots of null errors in the tests with that change and I had to revert on these. Maybe I need to update the tests? But it looked as if the structure changed for what GraphQL is looking for, so I reverted the changes.